core/vm/interpreter/interpreter.hpp

```diff
@@ -32,7 +32,7 @@ namespace fc::vm::interpreter {
 
   struct InterpreterCache {
     struct Key {
-      Key(const TipsetKey &tsk);
+      explicit Key(const TipsetKey &tsk);
```

Comment:
explicit constructor looks ugly.  
this struct was intended to avoid duplicating methods (has/get/set/remove) which can accept either `TipsetKey` or reuse allocated `Key` in subsequent calls (eg cached interpreter, not sure if there is any meaning in reuse).
may remove this struct and change `Key` usages to `TipsetKey`.